keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Non graceful shutdown
+authors:
+  - "@yastij"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+approvers:
+  - "@saad-ali"
+  - "@thockin"
+reviewers:
+  - "@andrewsykim"
+  - "@xing-yang"
+editor: Yassine Tijani
+creation-date: 2019-06-26
+last-updated: 2020-01-16
+status: implementable
+---
+
+# Non graceful shutdown
+
+This includes the Summary and Motivation sections.
+
+
+## Table of Contents
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+- [Implementation History](#implementation-history)
+- [Alternatives](#alternatives)
+  - [rely on the taint as a lock mechanism (taint and untaint by the same actor)](#rely-on-the-taint-as-a-lock-mechanism-taint-and-untaint-by-the-same-actor)
+<!-- /toc -->
+
```

Comment:
Need a section for Production Readiness Review.  The `feature enablement and rollback` section is required for alpha:

https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template#feature-enablement-and-rollback